SimCalorimetry/Configuration/python/hcalDigiSequence_cff.py

```diff
@@ -9,7 +9,8 @@
                                 +simHcalDigis
                                 *simHcalTTPDigis)
 
-_phase2_hcalDigiSequence = hcalDigiSequence.copyAndExclude([simHcalTriggerPrimitiveDigis,simHcalTTPDigis])
+#_phase2_hcalDigiSequence = hcalDigiSequence.copyAndExclude([simHcalTriggerPrimitiveDigis,simHcalTTPDigis])
```

Comment:
these lines can just be removed. Perhaps its worth a small PR with just this change as the first step?